docs/advanced.md

```diff
@@ -2,1231 +2,6 @@
 
 # Advanced fastlane
 
-# Passing Parameters
+This file is not used any more and its content was moved to subpages. Please follow the links to the content's new location.
 
```

Comment:
Can we do anything better than this? I know I have seen links out there to specific sections of this file using the HTML anchor.

Is it possible to 302 redirect to the correct location?

If not, can we keep the Markdown headings (which generate the anchors) and provide links to the appropriate section underneath?

Reply:
This PR is still WIP, I was planning on re-adding the headlines here with links to the new locations of the content.

302 redirect does not work, as the anchor is not submitted to the remote server but only handled in the browser. We could write some Javascript that reads this and knows where to redirect.

Reply:
This looks promising:
https://stackoverflow.com/a/21198129/252627
(only needs an additional check if a new URL was actually found before redirecting)

Reply:
This is now implemented and works pretty well.